Indonesian suku

bennylin opened the pull request:
Layout for Indonesian keyboard, for Indonesian language and Indonesian local languages.

Based on, and an improvement of US Basic layout.

keyman-server commented:
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

bennylin commented:
I don't know why my changes from /jawa branch is also included in this pull request. I've already committed the /jawa branch to origin, changed to master branch, rebased master to the latest version, create new branch indonesian-suku, made changes, committed to origin, and created this pull request. It should have the latest upstream version, right? Did I miss something?

bennylin commented:
https://github.com/keymanapp/keyboards/pull/1077
